tests/func/test_checkout.py

```diff
@@ -324,30 +324,25 @@ def test_checkout_relink(tmp_dir, dvc, link, link_test_func):
     dvc.cache.local.cache_types = [link]
 
     tmp_dir.dvc_gen({"dir": {"data": "text"}})
-    dvc.unprotect("dir/data")
-    assert not link_test_func("dir/data")
-
-    stats = dvc.checkout(["dir.dvc"], relink=True)
-    assert stats == empty_checkout
-    assert link_test_func("dir/data")
 
+    data_file = os.path.join("dir", "data")
 
-@pytest.mark.parametrize("link", ["hardlink", "symlink", "copy"])
-def test_checkout_relink_protected(tmp_dir, dvc, link):
```

Comment:
This was an old test from times before `protected` was default.

For now ignoring hardlink mode change on windows to move forward, but might reconsider later on.